README.md

```diff
@@ -18,6 +19,14 @@ it, simply add the following line to your Podfile:
 pod 'BonMot'
 ```
 
+## Installation with Carthage
+
+BonMot is also compatible with [Carthage](https://github.com/Carthage/Carthage). To install it, simply add the following line to your Cartfile:
+
+```
```

Comment:
Please tag this as `ogdl`. I'm not sure if GitHub flavored Markdown will render it, but it would be nice to have in case they add support.